packages/apps/graph/src/graph/resolvers/transaction-connection.ts

```diff
@@ -0,0 +1,84 @@
+import { prismaClient } from '@db/prisma-client';
+import type { ResolveCursorConnectionArgs } from '@pothos/plugin-relay';
+import { resolveCursorConnection } from '@pothos/plugin-relay';
+import type { Prisma } from '@prisma/client';
+import type { IContext } from '../builder';
+import { prismaTransactionsMapper } from '../mappers/transaction-mapper';
+
+export async function resolveTransactionConnection(
+  args: any,
+  context: IContext,
+  whereCondition: Prisma.TransactionWhereInput,
+) {
+  const totalCount = await prismaClient.transaction.count({
+    where: {
+      ...whereCondition,
+    },
+  });
+
+  const connection = await resolveCursorConnection(
+    {
+      args,
+      toCursor: (transaction) => {
```

Comment:
We could do this here:
```suggestion
      toCursor(transaction) {
```